dhis2/capture-app
# [TECH-672] Navigate to enrollment overview based on dataStore

https://jira.dhis2.org/browse/TECH-672

Migrate from old `navigateToTrackedEntityDashboard`-function, make API-call to fetch the dataStore/userDataStore and navigate the user to the correct dashboard.
Users can now try out the new enrollment dashboard with a config-json in the dataStorage with a "program-specific" enabling.  

Steps to test:
Make a POST-request to /api/dataStore/capture/useNewDashboard
Make API-body JSON with [programId]: "true/false"
The capture app should then navigate the user if the program is set to true.
There is also functionality to override the general config with a "per-user" enabling by making the same request as above in the /userDataStore-endpoint. This will then take precedence over the global dataStore-storage.
